sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/RecognizeLinkedEntityAsyncClient.java

```diff
@@ -115,14 +116,14 @@
      * which contains {@link RecognizeLinkedEntitiesResultCollection}.
      *
      * @param documents The list of documents to recognize linked entities for.
-     * @param options The {@link TextAnalyticsRequestOptions} request options.
+     * @param options The {@link RecognizeLinkedEntitiesOptions} request options.
      * @param context Additional context that is passed through the Http pipeline during the service call.
      *
      * @return A mono {@link Response} that contains {@link RecognizeLinkedEntitiesResultCollection}.
      */
     Mono<Response<RecognizeLinkedEntitiesResultCollection>>
         recognizeLinkedEntitiesBatchWithContext(Iterable<TextDocumentInput> documents,
-            TextAnalyticsRequestOptions options, Context context) {
+            RecognizeLinkedEntitiesOptions options, Context context) {
```

Comment:
i'm sure it's fine, but i just want to double check that this is not a breaking change for users

Reply:
This is an internally method. I add a new API to support `RecognizeLinkedEntitiesOptions`.
https://github.com/Azure/azure-sdk-for-java/pull/18984/files#diff-f772d27ac3cc6b57b1654f337f785208250b0a9191549584195354d3aa3d9ed7R735